signac/contrib/job.py

```diff
@@ -23,14 +23,22 @@
 
 
 class _sp_save_hook(object):
+    """
```

Comment:
Maybe something like "Force the job's statepoint to reset to whenever the statepoint is changed. Resetting a statepoint requires recomputing the hash and moving the folder"